opensrp-eusm/src/main/java/org/smartregister/eusm/presenter/EusmBaseDrawerPresenter.java

```diff
@@ -81,61 +86,67 @@ public void run() {
         }
     }
 
+    @Override
+    public void onOperationalAreaSelectorClicked(ArrayList<String> names) {
+        // Extract districts from selected regions
+        ArrayList<String> districts = new ArrayList<>();
+        try {
+            JSONArray locationHierarchy = new JSONArray(extractLocationHierarchy().first);
+            JSONArray regions = locationHierarchy.getJSONObject(0).getJSONArray(AppConstants.JsonForm.NODES);
+            for (int i = 0; i < regions.length(); i++) {
+                JSONObject location = regions.getJSONObject(i);
+                String locName = location.getString(TaskingConstants.CONFIGURATION.KEY);
+                if (names.contains(locName)) {
+                    names.remove(locName);
+                    if (location.has(AppConstants.JsonForm.NODES)) {
+                        JSONArray nodes = location.getJSONArray(AppConstants.JsonForm.NODES);
```

Comment:
Check for null on `nodes`.

Reply:
`nodes` will not be null as we already have a check at line `101` which make sure that the current location has nodes